db/mockDGen.js

```diff
@@ -1,10 +1,46 @@
+//generates and modifies data:
+const fs = require('fs');
+const mapData2 = require('./data/mapData2');
+const mapData = require('./data/mapData');
+const motherData = require('./data/MomaDummyData');
 const faker = require('faker');
 
+let id = 101;
+const modifiedData = titleData.map((item) => {
+  const randomNumber = Math.random();
+  item.id = id;
+  id += 1;
+  const result = [];
+  if (item.type) result.push(item.type);
+  if (randomNumber >= .5 || item.type === undefined) {
+    const randomIndex = Math.floor(Math.random() * 9);
+    const types = ['Pizza', 'Italian', 'American', 'Chinese', 'Italian', 'Mexican', 'Indian', 'French', 'Brunch'];
+    const randomType = types[randomIndex];
+    if (!result.includes(randomType)) {
+      result.push(randomType);
+    } 
+  }
+  item.type = result;
+  return item;
+});
+
+let id = 101
+const newData = mapData.map((item) => {
+  item.id = id;
+  id += 1;
+  return item;
+});
+
+fs.writeFile('db/data/mapData2.js', newData, (err) => {
```

Comment:
I think you're using fs to write a file with all your combined mock data, which then gets used to seed your db? Instead of writing another data file (and a fairly large one), what if you just seeded your database from inside this function? Or alternatively, you could push the data into an array, return the array, and then export this function so that you could call this function in another document to seed your data. There isn't really a reason to have the data saved in a file as well as in a database, especially if you're using a function to generate the data.

Reply:
I have a function in seedDB.js which is what seeds my database. I needed to use this function here and store the data in a file because Kevin is relying on the mock data I generated earlier and I wanted to modify that data slightly (add multiple types for each restaurant). Ultimately my seedDB.js module will use the 'momma data' that we had earlier made as a team, plus the map data that Kevin and I are jointly using.